src/catchup/ApplyBufferedLedgersWork.cpp

```diff
@@ -39,6 +41,7 @@ ApplyBufferedLedgersWork::onRun()
     auto& lm = mApp.getLedgerManager();
     if (!lm.hasBufferedLedger())
     {
+        mOnSuccess();
```

Comment:
A cleaner way would be to do this callback in designated the `onSuccess` function. This way, if this function changes to return WORK_SUCCESS in multiple places, `onSuccess` would properly handle it.

Reply:
done